calculation/services.py

```diff
@@ -22,14 +22,14 @@ def get_rule_details(class_name):
 
 
 def run_calculation_rules(instance, context, user):
-    list_results_calculation = []
     for calculation_rule in CALCULATION_RULES:
         result_signal = calculation_rule.signal_calculate_event.send(
             sender=instance.__class__.__name__, instance=instance, user=user, context=context
         )
         if result_signal:
-            list_results_calculation.extend(result_signal)
-    return list_results_calculation
+            return result_signal
```

Comment:
In case when CALCULATION_RULES have more than one entry only the first rule will be executed and result_signal for this rule will be returned. Is this expected behavior?

Reply:
As I wrote to you on slack - I defined that method according to the wiki page. I added the comment on page (I asked Patrick similar question and I have some doubts about this case too as you). I can modify it later when I got the answer for that question.